src/any-type-field/any-type-field.component.ts

```diff
@@ -54,7 +55,13 @@ export class AnyTypeFieldComponent {
   @Input() path: Array<any>;
   @Input() value: any;
 
-  constructor(public appGlobalsService: AppGlobalsService) { }
+
+  constructor(public changeDetectorRef: ChangeDetectorRef,
+    public appGlobalsService: AppGlobalsService) {
+    this.appGlobalsService.adminModeSubject.subscribe(adminMode => {
```

Comment:
Is this needed? Since you already markForCheck in primitive-field-component (which is a child of this component) this should be checked as well.